conference.js

```diff
@@ -184,29 +184,33 @@ function muteLocalVideo (muted) {
  * If requested show a thank you dialog before that.
  * If we have a close page enabled, redirect to it without
  * showing any other dialog.
- * @param {boolean} showThankYou whether we should show a thank you dialog
+ *
+ * @param {object} data Feedback data
+ * @param {boolean} data.showThankYou - whether we should show thank you dialog
+ * @param {boolean} data.feedbackSubmitted - whether feedback was submitted
  */
-function maybeRedirectToWelcomePage(showThankYou) {
-
+function maybeRedirectToWelcomePage(data) {
```

Comment:
If data is representing feedback data let's call the parameter feedbackData or feedbackOptions or something similar, this will make the code more readable.